pkgs/data/fonts/borg-sans-mono/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchzip }:
+
+let
+  version = "0.2.0";
+in
+fetchzip {
+  name = "borg-sans-mono-ttf-${version}";
+
+  url = "https://github.com/marnen/borg-sans-mono/files/107663/BorgSansMono.ttf.zip";
+  sha256 = "1gz4ab0smw76ih5cs2l3n92c77nv7ld5zghq42avjsfhxrc2n5ri";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    unzip -j $downloadedFile \*.ttf -d $out/share/fonts/truetype
+  '';
+
+  meta = with lib; {
+    description = "Droid Sans Mono Slashed + Hasklig-style ligatures!";
```

```suggestion
    description = "Droid Sans Mono Slashed + Hasklig-style ligatures";
```

contributing guidance doesn't want descriptions ending with `.`, I think that extends to other punctuation

---

Done, thx!